bin/oref0-determine-basal.js

```diff
@@ -58,20 +79,40 @@ if (!module.parent) {
             meal_data = JSON.parse(fs.readFileSync(meal_input, 'utf8'));
             console.error(JSON.stringify(meal_data));
         } catch (e) {
-            console.error("Optional feature Meal Assist not configured.");
+            var msg = {
+              msg: "Optional feature Meal Assist enabled, but could not read required meal data."
+            , file: meal_input
+            , error: e
+            };
+            console.error(msg.msg);
+            // console.log(JSON.stringify(msg));
+            errors.push(msg);
+            // process.exit(1);
         }
     }
     //if (meal_input) { meal_data = require(cwd + '/' + meal_input); }
 
     //console.error(autosens_input);
-    var autosens_data = { "ratio":1 };
-    if (typeof autosens_input !== 'undefined') {
+    var autosens_data = null;
+    if (autosens_input) {
+      // { "ratio":1 };
+      autosens_data = { "ratio": 1 };
+      if (autosens_input !== true && autosens_input.length) {
```

Comment:
For clarity I'd do `var autosens_enabled = params.autoSens` and `var autosens_input = autosens_enabled ? params._.slice...` instead of overloading this with booleans and strings...